.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: "weekly"
+- package-ecosystem: "cargo"
+  directory: "/"
```

Comment:
Does this include `vendor/fuser` create? we might prefer to pull the changes from upstream for that one.

Reply:
Oh, you might be right on that one.

I've disabled Cargo updates for now anyway (#347) since it was making breaking changes which we weren't expecting.